pkgs/profpatsch/nman/nman.rs

```diff
@@ -212,7 +212,7 @@ fn build_man_page<'a>(drv: DrvWithOutput, section: Option<&str>, page: &str, tem
     for dir in section_dirs {
         // separate "man" prefix from section indicator,
         // while validating the particular sub directory
-        match dir.to_str().map(|d| d.split_at(3)) {
+        match dir.to_str().filter(|d| d.len() > 3).map(|d| d.split_at(3)) {
```

Comment:
Apparently `/fr` `/de` and the like are for the translated versions of manpages oO

Reply:
I've gone down the path of `mandoc` and OpenBSD in denying that these should exist, but it'd be relatively simple to add a flag for that in the future. Not sure if is really worth it though, as not even all coreutils man pages are available translated and I don't suspect many people are using translated man pages.

Not sure if this is even a commendable GNU invention (probably).